code/game/objects/structures/crates_lockers/closets/secure/engineering.dm

```diff
@@ -91,10 +91,10 @@
 	new /obj/item/pipe_dispenser(src)
 	new /obj/item/storage/toolbox/mechanical(src)
 	new /obj/item/tank/internals/emergency_oxygen/engi(src)
-	new /obj/item/analyzer(src)

```

Comment:
is this because there's already one in the toolbox? undoc'd change

Reply:
i really need to stop doing undocumented changes aaaaaaaaaaa

yes cobby, it's pretty useless so i decided to remove it @ExcessiveUseOfCobblestone

Reply:
when I review I get notifs w/o need to ping btw

Reply:
but it took you 20 hrs to respond.....................

Reply:
eat / breathe / sleep

ss13